composer/helpers/v1/build

```diff
@@ -9,7 +9,16 @@ if [ -z "$install_dir" ]; then
 fi
 
 helpers_dir="$(dirname "${BASH_SOURCE[0]}")"
-cd "$helpers_dir"
+cp -r \
+  "$helpers_dir/bin" \
+  "$helpers_dir/src" \
+  "$helpers_dir/.php_cs" \
+  "$helpers_dir/composer.json" \
+  "$helpers_dir/composer.lock" \
+  "$helpers_dir/phpstan.neon" \
```

Comment:
This is linting configuration, I don't think we need it in the prod build?

Reply:
👍🏻  Good point, will remove

Reply:
It turns out we do actually use this as part of the build script, we run the install and then 

```
composer run lint -- --dry-run
composer run stan
```

It's slightly unexpected as it means we'd experience a breakage at image build time rather than in a CI test, but I'm going to leave it as-is for now.